### PR TITLE
Fixes a bug in PlayHead position in TimeRule when stretching

### PR DESCRIPTION
**Pull request by vmartos**

### CODE REVIEW

File: VAS.UI.Gtk2/UI/Component/Timeline.cs [hunk 1]
```diff
@@ -481,8 +481,12 @@ protected void HandleTimeruleSeek (Time pos, bool accurate, bool synchronous = f
 
 		void HandlePlayerTick (PlayerTickEvent e)
 		{
-			CurrentTime = e.Time;
 			relativeTime = e.RelativeTime;
+			if (isTimeLineEvent) {
```
Comment (ylatuya) [inline VAS.UI.Gtk2/UI/Component/Timeline.cs]: Why not always us CurrentTime = e.RelativeTime ?
The player controller already takes care of this logic.

Comment (ylatuya) [inline VAS.UI.Gtk2/UI/Component/Timeline.cs]: I don't understand what's the use of isTimelineEvent, is it used to determine if the timeline is streched or not?

File: VAS.Services/PlaylistManager.cs [hunk 2]
```diff
@@ -157,17 +157,10 @@ protected virtual void HandleLoadCameraEvent (LoadCameraEvent e)
...
 				}
-
-				if (e.CameraTlEvent.Stop != Player.FileSet.Duration) {
-					seekTime = seekTime >= e.CameraTlEvent.Stop ? e.CameraTlEvent.Stop : seekTime;
-					if (Player.Playing) {
-						Player.Stop ();
-					}
-				}
-
+				seekTime = seekTime > e.CameraTlEvent.Stop ? e.CameraTlEvent.Stop : seekTime;
```
Comment (ylatuya) [inline VAS.Services/PlaylistManager.cs]: I think you can use the Clamp extension method here:
seekTime = seekTime.Clamp (new Time(0), e.CameraTlEvent.Stop);

File: VAS.UI.Gtk2/UI/Component/Timeline.cs [hunk 3]
```diff
@@ -377,23 +374,12 @@ protected bool UpdateTime ()
...
-				if (isTimeLineEvent) {
-					timeline.CurrentTime = currentTime;
-					timerule.CurrentTime = currentTime;
-				} else {
-					timeline.CurrentTime = relativeTime;
-					timerule.CurrentTime = relativeTime;
-				}
-
+				timeline.CurrentTime = currentTime;
+				timerule.CurrentTime = currentTime;
```
Comment (ylatuya) [inline VAS.UI.Gtk2/UI/Component/Timeline.cs]: duplicated line

Comment (vmartos) [inline VAS.UI.Gtk2/UI/Component/Timeline.cs]: one is for timeline and the other is for the timerule

**Pull request merged**